test/integration/answer_test.rb

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
```

Comment:
I know it's copy-pasta but for Ruby > 2.0 there's no need to explicitly set the encoding to UTF-8 like this.